### PR TITLE
Only try to get AdditionalGroups if they are configured.

### PR DESCRIPTION
**Pull request by mrunalp**
Signed-off-by: Mrunal Patel <mrunalp@gmail.com>

### CODE REVIEW

**Comment by mrunalp** [comment]
@LK4D4 PTAL

**Comment by mrunalp** [comment]
@crosbymichael PTAL

**Comment by LK4D4** [comment]
LGTM

File: init_linux.go [hunk 1]
```diff
@@ -178,9 +178,12 @@ func setupUser(config *initConfig) error {
 		return err
 	}
 
-	addGroups, err := user.GetAdditionalGroupsPath(config.Config.AdditionalGroups, groupPath)
-	if err != nil {
-		return err
+	var addGroups []int
+	if config.Config.AdditionalGroups != nil {
```
Comment (crosbymichael) [inline init_linux.go]: wouldn't len() > 0 be better here?

Comment (mrunalp) [inline init_linux.go]: Fixed.

**Comment by dqminh** [comment]
LGTM

**Comment by mrunalp** [comment]
Thanks @dqminh 
@LK4D4 new tag for vendoring into docker? :)

**Pull request merged**